packages/vaadin-upload/theme/lumo/vaadin-upload-styles.js

```diff
@@ -75,7 +75,7 @@ registerStyles(
 
     [part='row'] {
       display: flex;
-      align-items: baseline;
+      align-items: center;
```

Comment:
For some reason, earlier the file content was aligned by `baseline`, but the prototype [aligns the content](https://github.com/web-padawan/a11y-vaadin-proto/blob/master/src/v-upload-file.js#L17) by `center`.

I decided to follow the prototype's approach and align the content by `center` as the content looks neater in this case.